index.html

```diff
@@ -1412,6 +1412,72 @@ <h3>Biometrics</h3>
     </section>
   </section>
 
+  <section class="informative">
+    <h2>Environmental and Ethical Considerations</h2>
+
+    <section class="informative">
+      <h3>Energy Usage and Environemntal Impacts</h3>
+      <p>
+        When implementing or utilizing a DID method, consideration should be given
+        to the environmental impacts of any underlying technologies.  For example
+        there are clear impacts from excess energy use from use of the "proof of work"
+        approaches in various blockchains or distributed ledgers.
+        Avoidance of excess energe use, particularly when that energy is sourced
+        from non-sustainable energy creation methods should be a best practice 
+        that serves the interest of all.
+      </p>
+
+      <p class="advisement">
+        Utilizing or authoring DID methods that rely on "proof of work" as part
+        of their required technical implementation or utilization should be highly
+        discouraged if not viewed as irresponsible.
```

Comment:
I suspect folks will object to this, see https://assets.ctfassets.net/2d5q1td6cyxq/5mRjc9X5LTXFFihIlTt7QK/e7bcba47217b60423a01a357e036105e/BCEI_White_Paper.pdf

Reply:
Agreed - I think there is room for debate on the subject, but the science is getting more clear on negative impacts from proof of work as well as being pretty measurable [cbeci](https://cbeci.org/)

Reply:
@OR13 see updates to this section